debug/org.eclipse.cdt.debug.application.doc/.project

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
You have accidentally restored/recreated a project that has been removed from CDT very recently.

Reply:
OMG, silly of me.
Removed now.